e2e/wallets.po.ts

```diff
@@ -280,14 +228,95 @@ export class WalletsPage {
     });
   }
 
+  checkThirdAddress() {
+    const lastRecord = element.all(by.css('.-record')).last();
+    return lastRecord.element(by.css('.address-column')).getText().then((address) => {
+      return address === 'rK1CMcqXjJ59H7XXs9xR3JZMejWs56FsmY';
+    });
+  }
+
   showPriceInformation() {
     browser.wait(ExpectedConditions.presenceOf(element(by.css('.balance p.dollars'))), 5000);
     return element(by.css('.balance p.dollars')).getText().then(text => {
-      return this._checkHeaderPriceFormat(text);
+      return this.checkHeaderPriceFormat(text);
+    });
+  }
+
+  openDeleteWalletDialog() {
+    return element.all(by.css('.btn-delete-wallet')).first().click().then(() => {
+      return element(by.css('app-confirmation')).isPresent();
+    });
+  }
+
+  cancelDeleteConfirmation() {
+    let walletCount = 0;
+    element.all(by.css('.-wallet')).count().then((count) => walletCount = count);
+
+    return element(by.buttonText('No')).click().then(() => {
+      return element.all(by.css('.-wallet')).count().then((count) => {
+        return count === walletCount;
+      });
+    });
+  }
+
+  deleteWallet() {
+    const walletNameToDelete = 'Test Create Name';
```

Comment:
This name is not used previously, so a wallet with this name will never be found, regardless of whether the delete operation was successful or not.

Reply:
Fixed.